org-timeblock.el

```diff
@@ -1413,6 +1415,30 @@ When called from Lisp, DATE should be a date as returned by
       (ot-show-context)
       (recenter))))
 
+(defun ot-clock-in ()
+  "Jump to the org heading of selected timeblock."
+  (interactive)
+  (unless (eq major-mode 'org-timeblock-mode)
+    (user-error "Not in *org-timeblock* buffer"))
+  (goto-char (point-min))
+  (search-forward "</rect>" nil t)
+  (when (re-search-forward (format "<rect .*? id=\"\\([^\"]+\\)\" fill=\"%s\"" ot-sel-block-color) nil t)
+    (when-let ((inhibit-read-only t)
+	             (id (match-string-no-properties 1))
+	             (m (cadr (seq-find (lambda (x) (string= (car x) id)) ot-data))))
```

Comment:
Single-letter variable names should generally be avoided.  Descriptive ones make code easier to read.  :)

Also, consider using a form like, `(cl-find id ot-data :key #'car :test #'equal)`.